addons/mrp/i18n/mrp.pot

```diff
@@ -1097,7 +1097,13 @@ msgstr ""
 #. module: mrp
 #. odoo-javascript
 #: code:addons/mrp/static/src/components/wo_list_view_dropdown/wo_list_view_dropdown.xml:0
-msgid "Block WC"
+msgid "Block"
+msgstr ""
```

Comment:
:skull: don't manually change stuff in the pot file in the future please. Export the pot file + compare the diff. Duplicated terms like this break the Transifex sync + the terms are unicode alphabetized